tests/Doctrine/Tests/DBAL/Platforms/SQLServerPlatformTest.php

```diff
@@ -58,4 +59,19 @@ public function getModifyLimitQueries()
         );
     }
 
+    public function testGetDefaultValueDeclarationSQLForDateType()
+    {
+        $currentDateSql = $this->_platform->getCurrentDateSQL();
+	    foreach (array('date', 'date_immutable') as $type) {
+		    $field = array(
+			    'type'    => Type::getType($type),
+			    'default' => $currentDateSql,
+		    );
+		    
+		    $this->assertEquals(
```

Comment:
Please use `self::assert*` instead, and it would be better to use `assertSame()` in this case